jbang.ps1is not a correct application of system Win32

**Describe the bug**
After fresh installation of jbang when I try to run quarkus-mcp-servers I get the error:

```
❯ jbang jdbc@quarkiverse/quarkus-mcp-servers
java.io.IOException: Cannot run program "C:\Users\janwa\.jbang\bin\jbang.ps1": CreateProcess error=193, %1 nie jest prawidłową aplikacją systemu Win32
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
        at jdbc.call(mcpjdbc.java:121)
        at jdbc.call(mcpjdbc.java:23)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at jdbc.main(mcpjdbc.java:44)
Caused by: java.io.IOException: CreateProcess error=193, %1 nie jest prawidłową aplikacją systemu Win32
        at java.base/java.lang.ProcessImpl.create(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
        ... 12 more
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install with `iex "& { $(iwr -useb https://ps.jbang.dev) } app setup"`
2. jbang jdbc@quarkiverse/quarkus-mcp-servers

Did nothing else.
```
❯ java --version
java 17.0.12 2024-07-16 LTS
Java(TM) SE Runtime Environment (build 17.0.12+8-LTS-286)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.12+8-LTS-286, mixed mode, sharing)
❯ jbang version
0.126.0
```

**Expected behavior**
I expected it to launch an empty db like in the [general usage](https://github.com/quarkiverse/quarkus-mcp-servers/tree/main/jdbc#general-usage)

**JBang version**
```
[jbang] [0:329] jbang version 0.126.0
Cache: C:\Users\janwa\.jbang\cache
Config: C:\Users\janwa\.jbang
[jbang] [2025-05-13 16:26:03] X509Certificate: Alg:SHA256withRSA, Serial:570a119742c4e3cc, Subject:CN=Actalis Authentication Root CA, O=Actalis S.p.A./03358520967, L=Milan, C=IT, Issuer:CN=Actalis Authentication Root CA, O=Actalis S.p.A./03358520967, L=Milan, C=IT, Key type:RSA, Length:4096, Cert Id:1729119956, Valid from:22.09.2011, 13:22, Valid until:22.09.2030, 13:22
[jbang] [2025-05-13 16:26:03] X509Certificate: Alg:SHA1withRSA, Serial:1, Subject:CN=AddTrust External CA Root, OU=AddTrust External TTP Network, O=AddTrust AB, C=SE, Issuer:CN=AddTrust External CA Root, OU=AddTrust External TTP Network, O=AddTrust AB, C=SE, Key type:RSA, Length:2048, Cert Id:-326352672, Valid from:30.05.2000, 12:48, Valid until:30.05.2020, 12:48
[jbang] [2025-05-13 16:26:03] X509Certificate: Alg:SHA1withRSA, Serial:1, Subject:CN=AddTrust Qualified CA Root, OU=AddTrust TTP Network, O=AddTrust AB, C=SE, Issuer:CN=AddTrust Qualified CA Root, OU=AddTrust TTP Network, O=AddTrust AB, C=SE, Key type:RSA, Length:2048, Cert Id:607365522, Valid from:30.05.2000, 12:44, Valid until:30.05.2020, 12:44
[jbang] [2025-05-13 16:26:03] X509Certificate: Alg:SHA256withRSA, Serial:7777062726a9b17c, Subject:CN=AffirmTrust Commercial, O=AffirmTrust, C=US, Issuer:CN=AffirmTrust Commercial, O=AffirmTrust, C=US, Key type:RSA, Length:2048, Cert Id:630485644, Valid from:29.01.2010, 15:06, Valid until:31.12.2030, 15:06
[jbang] [2025-05-13 16:26:03] X509Certificate: Alg:SHA1withRSA, Serial:7c4f04391cd4992d, Subject:CN=AffirmTrust Networking, O=AffirmTrust, C=US, Issuer:CN=AffirmTrust Networking, O=AffirmTrust, C=US, Key type:RSA, Length:2048, Cert Id:651670175, Valid from:29.01.2010, 15:08, Valid until:31.12.2030, 15:08
[jbang] [2025-05-13 16:26:03] X509Certificate: Alg:SHA384withRSA, Serial:6d8c1446b1a60aee, Subject:CN=AffirmTrust Premium, O=AffirmTrust, C=US, Issuer:CN=AffirmTrust Premium, O=AffirmTrust, C=US, Key type:RSA, Length:4096, Cert Id:-2130283955, Valid from:29.01.2010, 15:10, Valid until:31.12.2040, 15:10
[jbang] [2025-05-13 16:26:03] X509Certificate: Alg:SHA384withECDSA, Serial:7497258ac73f7a54, Subject:CN=AffirmTrust Premium ECC, O=AffirmTrust, C=US, Issuer:CN=AffirmTrust Premium ECC, O=AffirmTrust, C=US, Key type:EC, Length:384, Cert Id:-2080363786, Valid from:29.01.2010, 15:20, Valid until:31.12.2040, 15:20
[jbang] [2025-05-13 16:26:04] X509Certificate: Alg:SHA256withRSA, Serial:66c9fcf99bf8c0a39e2f0788a43e696365bca, Subject:CN=Amazon Root CA 1, O=Amazon, C=US, Issuer:CN=Amazon Root CA 1, O=Amazon, C=US, Key type:RSA, Length:2048, Cert Id:-1472444962, Valid from:26.05.2015, 02:00, Valid until:17.01.2038, 01:00
[jbang] [2025-05-13 16:26:04] X509Certificate: Alg:SHA384withRSA, Serial:66c9fd29635869f0a0fe58678f85b26bb8a37, Subject:CN=Amazon Root CA 2, O=Amazon, C=US, Issuer:CN=Amazon Root CA 2, O=Amazon, C=US, Key type:RSA, Length:4096, Cert Id:-1766132388, Valid from:26.05.2015, 02:00, Valid until:26.05.2040, 02:00
[jbang] [2025-05-13 16:26:04] X509Certificate: Alg:SHA256withECDSA, Serial:66c9fd5749736663f3b0b9ad9e89e7603f24a, Subject:CN=Amazon Root CA 3, O=Amazon, C=US, Issuer:CN=Amazon Root CA 3, O=Amazon, C=US, Key type:EC, Length:256, Cert Id:-1562287523, Valid from:26.05.2015, 02:00, Valid until:26.05.2040, 02:00
Repository: C:\Users\janwa\.m2\repository
Java: C:\Program Files\Java\jdk-17 [17.0.12]
[jbang] [2025-05-13 16:26:04] X509Certificate: Alg:SHA384withECDSA, Serial:66c9fd7c1bb104c2943e5717b7b2cc81ac10e, Subject:CN=Amazon Root CA 4, O=Amazon, C=US, Issuer:CN=Amazon Root CA 4, O=Amazon, C=US, Key type:EC, Length:384, Cert Id:-1654272513, Valid from:26.05.2015, 02:00, Valid until:26.05.2040, 02:00
OS: windows
Arch: x64
Shell: powershell
[jbang] [2025-05-13 16:26:04] X509Certificate: Alg:SHA1withRSA, Serial:20000b9, Subject:CN=Baltimore CyberTrust Root, OU=CyberTrust, O=Baltimore, C=IE, Issuer:CN=Baltimore CyberTrust Root, OU=CyberTrust, O=Baltimore, C=IE, Key type:RSA, Length:2048, Cert Id:1425294543, Valid from:12.05.2000, 20:46, Valid until:13.05.2025, 01:59
[jbang] [2025-05-13 16:26:04] X509Certificate: Alg:SHA256withRSA, Serial:2, Subject:CN=Buypass Class 2 Root CA, O=Buypass AS-983163327, C=NO, Issuer:CN=Buypass Class 2 Root CA, O=Buypass AS-983163327, C=NO, Key type:RSA, Length:4096, Cert Id:969960563, Valid from:26.10.2010, 10:38, Valid until:26.10.2040, 10:38
0.126.0
```

**Additional context**
Add any other context about the problem here.


Prior to this I posted an issue at jbangdev repo as I thought it was an issue with jbang itself but they concluded it was an error with quarkus [link](https://github.com/jbangdev/jbang/issues/2015#event-17643428298)

it's a clean install with
```
java 17.0.12 2024-07-16 LTS
Java(TM) SE Runtime Environment (build 17.0.12+8-LTS-286)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.12+8-LTS-286, mixed mode, sharing)
```
using standard builtin powershell (with oh my posh)

--- a/jdbc/.scripts/mcpjdbc.java
+++ b/jdbc/.scripts/mcpjdbc.java
@@ -77,6 +77,7 @@ class jdbc implements Callable<Integer> {
 
         // use the jbang command from env or assume on path
         String jbangcmd = System.getenv("JBANG_LAUNCH_CMD");
+        
         if (jbangcmd == null) {
             String os = System.getProperty("os.name").toLowerCase();
             if (os.contains("win")) {
@@ -84,6 +85,10 @@ class jdbc implements Callable<Integer> {
             } else {
                 jbangcmd = "jbang";
             }
+        } else if(jbangcmd.endsWith(".ps1")) {
+            //dumb hack to avoid .ps1 files on windows
+            //https://github.com/quarkiverse/quarkus-mcp-servers/issues/65
+            jbangcmd = jbangcmd.substring(0, jbangcmd.length() - 4) + ".cmd";
         }
 
         command.add(jbangcmd);

